hash/msx1_cart.xml

```diff
@@ -10146,6 +10181,25 @@ kept for now until finding out what those bytes affect...
 		</part>
 	</software>
 
+	<software name="printlab">
+		<description>Print Lab (Japan)</description>
+		<year>1985</year>
+		<publisher>Sony</publisher>
+		<info name="alt_title" value="プリンターソフトプリントラボ"/>
```

Comment:
The box art and tape label say “Printer Soft Print Lab”, as does the Japanese title you’ve put here – the title in the description should match.  See pictures: https://www.generation-msx.nl/software/hal-laboratory/print-lab/release/628/

Reply:
Yes, that makes more sense. I have changed it to 'Printer Soft Print Lab'